src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -279,6 +310,11 @@ export class ItemByIdComponent implements OnDestroy, BeforeUnloadComponent, Pend
 
   showItemThreadWidget = !!appConfig.forumServerUrl;
 
+  @HostListener('window:resize')
+  resize(): void {
+    this.contentContainerTop = this.contentContainer?.nativeElement.offsetTop || 0;
+  }
```

Comment:
Cannot you put it in the `merge` you have added instead of repeating the same here? 
(with an event such as `fromEvent(window, 'resize')`)

Reply:
Fixed